apps/lukso-manager/src/runner/config.go

```diff
@@ -0,0 +1,33 @@
+package runner
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/go-yaml/yaml"
+)
+
+type NetworkConfig struct {
+	GENESISTIME       int    `yaml:"GENESIS_TIME"`
+	CHAINID           int    `yaml:"CHAIN_ID"`
+	NETWORKID         int    `yaml:"NETWORK_ID"`
+	FORKCHOICE        int    `yaml:"FORK_CHOICE"`
+	PANDORABOOTNODES  string `yaml:"PANDORA_BOOTNODES"`
+	VANGUARDBOOTNODES string `yaml:"VANGUARD_BOOTNODES"`
+}
+
+func ReadConfig(network string) (*NetworkConfig, error) {
+	fileName := "/opt/lukso/networks/" + network + "/config/network-config.yaml"
+	buf, err := ioutil.ReadFile(fileName)
```

Comment:
defer buf.close()

Reply:
<img width="905" alt="Screenshot 2021-11-08 at 15 21 01" src="https://user-images.githubusercontent.com/798709/140758485-8a2ef132-2016-4dbe-a895-3c6af0ee02df.png">

I suppose the variable naming was bad.